source-browser.html

```diff
@@ -132,7 +142,8 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           $('#topics').append(
             $('<div class="topic flex-auto">').append(
               $('<div>').append(
-                $('<h2 class="inline-block m0">').text(name),
+                $('<h2 class="inline-block m0 relative topic-heading">', {id: `${name}-space`}),
```

Comment:
Wait. I didn't realize you had two of these elements... This is super hacky :( Have you tried either of the solutions at https://stackoverflow.com/questions/4086107/fixed-page-header-overlaps-in-page-anchors ?

Reply:
I have, actually.   They haven't worked especially well: the main problem is that the headers at the top don't work.